ci-operator/config/openshift/osde2e/openshift-osde2e-master.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  to: osde2e
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-4.1
+  commands: |
+    go get -u github.com/Masterminds/glide
```

Comment:
Generally these sort of things would be in `binary_build_commands` and you'd run this test `from: bin` so that any tests that need the binaries could re-use that build work instead doing it themselves.

Reply:
Cool, thanks for the heads up. I'll look into this.